{{cookiecutter.github_repository}}/.env.sample

```diff
@@ -10,6 +10,12 @@
 # SITE_NAME='your site name'
 # ALLOWED_HOSTS=youdomain.com
 # DJANGO_SECURE_HSTS_SECONDS=60
+#CORS_ORIGIN_WHITELIST = (
```

Comment:
This should be in single line and separated by commas and should be available only if add_django_cors_headers is selected by user.

```
{%- if cookiecutter.add_django_cors_headers.lower() == 'y' %}
CORS_ORIGIN_WHITELIST=localhost:8000,127.0.0.1:9000,example.com
{%- endif %}
```